pywbem/cim_obj.py

```diff
@@ -2853,6 +2855,148 @@ def tomof(self, indent=0, maxline=MAX_MOF_LINE):
+        optionally set the path component of the CIMInstance.
+        No CIMProperty qualifiers are included in the created instance and the
+        `class_origin` attribute is transfered from the class only if the
+        `include_class_origin` parameter is `True`.
+
+        Parameters:
+          klass (:class:`pywbem:CIMClass`)
+            CIMClass from which the instance will be constructed.  This class
+            must include qualifiers and should include properties from any
+            superclasses to be sure it includes all properties that are to be
+            built into the instance, in particular any key properties if the
+            `include+path` parameter is `True`. See
+            :meth:`~pywbem.CIMInstanceName.from_class` for further requirements
+            on the class.
+
+          namespace (:term:`string`):
+            Namespace to be included in the path component of the returned
+            CIMInstance if `include_path` parameter is `True`.
+
+          property_values (:class:`py:dict` or `NocaseDict`_):
+            Dictionary containing name/value pairs where the names are the
+            names of properties in the class and the properties are the
+            property values to be set into the instance properties. The values
+            must match the type defined for the property in the class. If a
+            property is in the property_values dictionary but not in the class
+            a ValueError exception is raised.
+
+          include_missing_properties (:class:`py:bool`):
+            Determines if properties not in the `property_values` parameter are
+            included in the instance.
+
+            If `True` all properties from the class are included in the new
+            instance including those not defined in `property_values` parameter
+            with any value from the class. This is the `Default`.
+
+            If `False` only properties in the `property_values` parameter are
+            included in the new instance.
+
+         include_class_origin  (:class:`py:bool`):
+            Determines if class origin information from the class is included
+            in the returned instance. Default is `False`.
+
+            If `None` or `False`, class origin information is not included.
+
+            If `True`, class origin information is included.
+
+          include_path (:class:`py:bool`:):
+            If `True` the :class:`~pywbem.CIMInstanceName` path is inserted
+            into the new instance based on properties in the new instance.  If
+            `strict` all key properties must be in the instance.
+
+          strict (:class:`py:bool`:):
+            This parameter is only used if the `include_path parameter is
+            `True`.
+
+            If `True` all key properties from the class must exist the instance
+            so that a correct path can be built. Default is `False` or an
+            ValueError exception is raised. This is the default.
```

Comment:
On "This is the default": Normally, we don't repeat the default of the method argument.

Reply:
Removed